core/jazz_cloud-logs-streamer/components/config.js

```diff
@@ -41,6 +41,8 @@ var getStageConfig = (event, context) => {
       stage = 'stg';
     } else if (fnName.endsWith('prod')) {
       stage = 'prod';
+    } else {
```

Comment:
Adding our comments thread from the other PR here:

Can you provide context as to what scenarios is this needed in? Introducing this will break environment management.

 @suryam0801
suryam0801 2 days ago Author
When writing tests for the buildRequest and post function, end of the function name used is "-test". This test file contains the ES_ENDPOINT: "abc.us-east-1.es.amazonaws.com" which is suitable to run the tests for the post and buildRequest functions

 @suryajak
suryajak 1 minute ago Owner
The tests should fall into this other if block at the top here: https://github.com/suryajak/jazz/pull/8/files#diff-4c6e3fdc46524adc9896feae482fa946R32.

Introducing an else block here opens up the code for a category of errors. For changes you are trying to introduce in the actual code always think of the impact area. One of the ways is to follow all the places where it is being used and see how that affect the callers.